### PR TITLE
Projects popup

### PR DESCRIPTION
**Pull request by Artbsc1992**
# Coding partner: @Danie12345 
### Achievements.
-Build new HTML elements with javascript.
-Store data objects within an array.
-Make popup responsive and stylish.

### CODE REVIEW

**Comment by Artbsc1992** [comment]
Hello, @chaw-bot I had to do a last time commit sorry for that could you please review my code again?

File: index.js [hunk 1]
```diff
@@ -12,4 +13,145 @@ navItems.forEach((navItem) => {
...
+const cardsArr = [
+  {
+    name: 'card2',
+    image: 'desktop-img/Project1.jpg',
+    title: 'Professional Art Printing Data',
+    text: 'A daily selection of privately personalized reads; no accounts or sign-ups required; has been the industry\'s standard',
+    buttons: {
+      langs: ['HTML', 'Bootstrap', 'Ruby on Rails'],
+      other: [['See Live', '', 'Img/see-project-icon.svg'], ['See Source', '', 'Img/source-icon.svg']],
+    },
```
Comment (sja-thedude) [inline index.js:27]: - I have added a video below which shows that you have not added a popup for multi-post stories and links for see project and see source.

https://user-images.githubusercontent.com/70269507/162087327-c9be7230-1e91-4554-8003-81ffb6aeea63.mov

- [ ] Kindly add links to the See Live and See Source as its shows the project details you made.
- [ ] Kindly add the popup for the mobile version and desktop version for Multi Post Stories as per the Figma design given below.
- Desktop popup:
Screenshot 2022-04-07 at 1 47 23 AM
- Mobile Popup:
Screenshot 2022-04-07 at 1 45 45 AM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Artbsc1992, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good commit messages
✔️ All linters are passing
✔️ Good PR Title and description

## Required 
- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — COMMENTED** [review]
Hello @Artbsc1992  👋,

_This is an additional review as a result of the second opinion request._

- Firstly i can see that all the necessary popup sections were created dynamically ( name, description, featured image, technologies, link to live version, link to source.) 
image

- Secondly, note that all see projects buttons should lead to a popup so as the reviewer requested kindly fix it 🙏 

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away :)

Keep rocking 🚀!

**Review by Meltrust — APPROVED** [review]
Hi @Artbsc1992,


## Wow, you did it 🎉
Brilliant
## Thank you for implementing the changes requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**